recipes/neo4j-python-driver/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name='neo4j-python-driver' %}
+{% set version='1.1.0rc1' %}
```

Comment:
I'd love to have recipes built for 1.1.0rc1 *and* 1.3.1, is this possible? There was some discussion of how to do this for *bioconda* here: https://github.com/bioconda/bioconda-recipes/issues/232 but does conda-forge have a mechanism for this?